_posts/2017-10-23-upgrade-to-rails-3-2.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Upgrade Rails from 3.1 to 3.2"
+date: 2017-10-23 10:00:00
```

Comment:
@lubc Once this post is approved please update this to the approval date and regenerate the HTML.